python/ray/tune/examples/cifar10_pytorch.py

```diff
@@ -173,11 +189,11 @@ def main(num_samples=10, max_num_epochs=10):
         grace_period=1,
         reduction_factor=2)
     reporter = CLIReporter(
-        parameter_columns=["l1", "l2", "lr", "batch_size"],
+        # parameter_columns=["l1", "l2", "lr", "batch_size"],
         metric_columns=["loss", "accuracy", "training_iteration"])
     result = tune.run(
         train_cifar,
-        resources_per_trial={"cpu": 1},
+        resources_per_trial={"cpu": 8, "gpu": gpus_per_trial},
```

Comment:
oh i think this might break on CI since GPUs are not available on CI; maybe we can pass this in only if smoke test is false?

CI i think generally has 2 CPUs, 0 GPUs total.